packages/core/src/scp-cf/destination.ts

```diff
@@ -152,6 +172,7 @@ export interface DestinationConfiguration {
   clientId?: string;
   clientSecret?: string;
   SystemUser?: string;
+  Type?: string;
```

Comment:
could we use a better type then string? A union type with the possble values like `HTTP`, `RFC` etc.

Reply:
I considered that too, but given, that we ignore all values except for 'HTTP', I thought this might be overkill. What do you think?

Reply:
I considered that, but initially decided to not do that, because I thought it might be overkill and I had to adjust some types in the tests for it. But now that you also think this might be better, I added it.